api/sshuser.go

```diff
@@ -150,7 +150,7 @@ func (s *SigningService) PostUserSSHCertificate(ctx context.Context, request *pr
 	}
 	respCh := make(chan resp)
 	go func() {
-		data, err := s.SignSSHCert(reqCtx, s.RequestChan[config.SSHUserCertEndpoint], cert, request.KeyMeta.Identifier, request.Priority)
+		data, err := s.SignSSHCert(reqCtx, s.RequestChan[config.SSHUserCertEndpoint], cert, request.KeyMeta.Identifier, request.Priority, s.RequestTimeout[config.SSHUserCertEndpoint])
```

Comment:
The timeout should have been set into reqCtx already, do we still need to pass the timeout to signer again?

Reply:
There are cases like [this](https://github.com/theparanoids/crypki/blob/main/api/blob.go#L111) which still use the context passed by the client which does not have a deadline/timeout. Once we over write all client context with context with timeout passed by the user (`reqCtx`), we can keep the check in `getRemainingRequestTime`. Once blob.go is also updated, we need not pass the timeout in signer & use default instead.